src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/TableBuilderController.cs

```diff
@@ -121,6 +121,25 @@ public async Task<ActionResult<TableBuilderResultViewModel>> QueryForTableBuilde
             return actionResult;
         }
 
+        [HttpGet("tablebuilder/release/data-block/{dataBlockParentId:guid}")]
+        public async Task<ActionResult<Dictionary<string, List<LocationAttributeViewModel>>>> QueryForTableBuilderWithGeoJsonResult(
```

Comment:
Does this being separate from the main query that brings back table data mean that we're having to issue multiple table queries per data block?  One for the table data, another for each boundary level as it's selected?

Reason I ask is that this could potentially become expensive on large tables, so we'll have to consider performance if I've got the right end of the stick!

Reply:
As an alternative, if this is issued as another query after the 1st query for table data (where we'd then know which Locations will appear in the results), could we instead post an array of Location Ids that we want boundary data for here rather than needing to issue a 2nd table query?

Reply:
Alternative sounds good to me, but the ticket specified endpoint arguments of "boundary level id and data block id" - I don't recall if it was considered as an option and thrown out, or missed as a potential solution - I'll see if anyone can remember, or if they agree it should be amended to how you suggest.

Reply:
I think when this was discussed the preference was for a GET request so it could be cached. We wouldn't necessarily be able to fit all the required ids in a GET url so it was decided that using the data block id and letting the back end figure out what geojson it needed to return would be best.